data-raw/process_data_sets.R

```diff
@@ -272,3 +272,18 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+
+
+#-------------------------------------------------------------------------------
+# Coffee quality data
+# Google Sheet here: https://docs.google.com/spreadsheets/d/1fscb1AbsSXWqqws-hhWAZfT580ms_FvOkBBQsJwozSY/edit
+# Original Github source: https://github.com/jldbc/coffee-quality-database
+#-------------------------------------------------------------------------------
+
+coffee_quality <- 
+  "https://wjhopper.github.io/SDS-201/data/coffee_ratings.csv" %>%
+  readr::read_csv()
```

Comment:
You don't need to explicitly call `readr::` since we load the tidyverse in this in R script.

Reply:
Addressed in commit [c450616](https://github.com/beanumber/moderndive/pull/13/commits/c4506166a6e2274a7335876961850f908f455a2c)